examples/volumes/azure_file/README.md

```diff
@@ -13,6 +13,7 @@ Then create a Pod using the volume spec based on [azure](azure.yaml).
 In the pod, you need to provide the following information:
 
 - *secretName*:  the name of the secret that contains both Azure storage account name and key.
+- *secretNamespace*:  the namespace of the secret. Default is the same namespace as the pod.
```

Comment:
this file is moved to https://github.com/kubernetes/examples. please update it there.